vendor/signify-osx/src/CVS/Entries

```diff
@@ -0,0 +1 @@
+D
```

Comment:
Might want to add the CVS stuff to .gitignore.

Reply:
Good idea, yes.